### PR TITLE
Add defaultSeverity config option; closes #1475

### LINKED ISSUE #1475: Add config property to change default severity
See discussion around https://github.com/stylelint/stylelint-config-standard/issues/23#issuecomment-226451861.

```js
{
  extends: "stylelint-config-standard",
  defaultSeverity: "warning"
  rules: {
    "color-no-invalid-hex": [ true, { severity: "error" } ]
  }
}
```

I think this may be pretty straightforward to implement.

cc @thomas-darling

### PR DESCRIPTION
**Pull request by davidtheclark**
Ref #1475

### CODE REVIEW

**Comment by evilebottnawi** [comment]
LGTM

**Comment by jeddy3** [comment]
That was simple :) LGTM. Feel free to merge on green.

File: docs/user-guide/configuration.md [hunk 1]
```diff
@@ -227,6 +227,13 @@ If the globs are absolute paths, they are used as is. If they are relative, they
 
 The `ignoreFiles` property is stripped from extended configs: only the root-level config can ignore files.
 
+### `defaultSeverity`
+
+The default severity level for all rules that have do not have a severity specified in their secondary options. The available values for `severity` are:
```
Comment (m-allanson) [inline docs/user-guide/configuration.md]: There's an extra 'have' here

Comment (jeddy3) [inline docs/user-guide/configuration.md]: It's nice to have so many eyes looking at things :)

**Pull request merged**